src/fields.json

```diff
@@ -1,275 +1,275 @@
 {
 	"list" : [
 		{
-			"label": "name",
+			"labelI18nKey": "name",
```

Comment:
I understand adding the `labelI18nKey` to each of these, but is there a reason for removing the `label` key?

Reply:
Further investigation, despite the semantic ambiguity, we could just use the `class` key for what you're using `labelI18Key` for. I included them to be valid HTML classes, but maybe it would be better to treat them (and possibly rename them) as slugs. What do you think?

Reply:
the key name is kinda arbitrary, I just changed it to "labelI18nKey" because it seemed like a decent description of what it is. Removing "label" was to tell future us that we shouldn't be hard-coding english nouns into the data, instead we should always use an i18n key/token for things like that.

I assumed that "class" was to become a CSS class, say if they were going to be styled visually in some fashion.

Reply:
Ok. I think you've convinced me. I'll make that change.

Reply:
You were totally right. This is much clearer. Thanks for your work on it. I put the strings in a separate localized-strings.json file so that it's a little easier for translators to contribute.